Lib/asyncio/events.py

```diff
@@ -418,6 +418,20 @@ async def create_unix_server(
         """
         raise NotImplementedError
 
+    async def connect_accepted_socket(
+            self, protocol_factory, sock,
+            *, ssl=None,
+            ssl_handshake_timeout=None):
+        """Handle an accepted connection.
+
+        This is used by servers that accept connections outside of
+        asyncio but that use asyncio to handle connections.
```

Comment:
I'm aware that the typo was present in the original docstring, but this seems like a good opportunity to fix it while it's being moved.

```suggestion
        asyncio, but use asyncio to handle connections.
```